### DISCUSSION TITLE
Conv parameters

### DISCUSSION THREAD

**Discussion by Bestupidas** [open-mmlab/mmdetection#9730] [Q&A] [answered]
code in mmdet/models/utils/res_layer.pys shows below(line 52~58):
class ResLayer(Sequential):
      ...
        if stride != 1 or inplanes != planes * block.expansion:
            downsample = []
            conv_stride = stride
     ...
            downsample.extend([
                build_conv_layer(
                    conv_cfg,
                    inplanes,
                    planes * block.expansion,
                    kernel_size=1,
                    stride=conv_stride,
                    bias=False),
                build_norm_layer(norm_cfg, planes * block.expansion)[1]
            ])
            downsample = nn.Sequential(*downsample)

my question:
 the conv layer  is built with kernel_size=1, stride=2 when conv_stride is 2. why conv stride is bigger than 1 when kener size is 1. is it right or has something carefully designed for specified purpose  ?  hope for reply

**Comment by ZwwWayne** [comment, marked as answer]
It is a convention when using the official implementation of ResNet in acedamia. In practice, using a larger kernel size (e.g., 3) or pooling layer could yield better performance.

**Reply by Bestupidas** [reply]
thanks !  i  will do an training experiment to check the modification from kernel size 1 to 3 to see its affects for map in dections.
